dolphinscheduler-master/src/test/java/org/apache/dolphinscheduler/server/master/integration/WorkflowTestCaseContextFactory.java

```diff
@@ -92,6 +92,8 @@ public WorkflowTestCaseContext initializeContextFromYaml(final String yamlPath)
         initializeTaskRelationsToDB(workflowTestCaseContext.getTaskRelations());
         if (workflowTestCaseContext.getWorkflowInstance() != null) {
             initializeWorkflowInstanceToDB(workflowTestCaseContext.getWorkflowInstance());
+        } else {
+            initializeWorkflowInstancesToDB(workflowTestCaseContext.getWorkflowInstances());
```

Comment:
```suggestion
        } 
        if(workflowTestCaseContext.getWorkflowInstances() != null) {
            initializeWorkflowInstancesToDB(workflowTestCaseContext.getWorkflowInstances());
        }   
```

Reply:
It's better to directly remove the `workflowinstance` in yaml, and only provide `workflowInstances`

Reply:
ok

Reply:
submitted, changed a lot test files.